onwuvic/MyDiary
# #159316611 Setup Postgre Database

#### What does this PR do?
Set up Postgre Database

#### Description of Task to be completed?
- Create entries.sql and users.sql files
- Set up database URL
- Set up env file

#### How should this be manually tested?
 After cloning the repo, cd into the directory. RUN 'npm install' to install dependencies.
Create the necessary database tables in src/migrations folder in your Postgre Admin.
 RUN 'npm run dev' your app should be connected to the database.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- Story type - Chores
- Story title - Set up Postgres database that persists user data
- Story id -  #159316611

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
